Documentation/ceph-cluster-crd.md

```diff
@@ -45,12 +45,88 @@ spec:
 ## PVC-based Cluster
 
 In a "PVC-based cluster", the Ceph persistent data is stored on volumes requested from a storage class of your choice.
-This type of cluster is recommended in a cloud environment where volumes can be dynamically created and also
+This type of cluster is recommended to use with CSI drivers which support dynamic volume provisioning and also in clusters where a local PV provisioner is available.
 in clusters where a local PV provisioner is available.
 
 > **NOTE**: Kubernetes version 1.13.0 or greater is required to provision OSDs on PVCs.
 
 ```yaml
```

Comment:
I'd like to keep the examples at the top of this page simple, and have more complex examples later in the doc. What about this?
- This example should match the cluster-on-pvc.yaml example for aws
- We should add a new section later in the doc for this example that creates local PVs and uses a local provisioner